src/app/public/modules/overlay/overlay-config.ts

```diff
@@ -25,4 +25,9 @@ export interface SkyOverlayConfig {
    */
   showBackdrop?: boolean;
 
+  /**
+   * Specifies classes to be added to the overlay element.
+   */
+  additionalClasses?: string;
```

Comment:
The overlay consists of several elements, so let's be more specific with the property name. Something like `wrapperClass`? (We may have similar properties in the future: `contentClass`, `backdropClass`.)
```
/**
 * Specifies extra CSS classes to be added to the overlay's wrapper element.
 */
wrapperClass?: string;
```
Naming convention inspired by Angular Material (see `backdropClass` and `panelClass`): https://material.angular.io/components/bottom-sheet/api#MatBottomSheetConfig